index.html

```diff
@@ -7749,24 +7737,24 @@
 		}
 
 		chatbody.innerHTML = newbodystr;
+	}
 
-		if(pending_response_id=="")
-		{
-			document.getElementById("chatistyping").classList.add("hidden");
-		}else{
-			document.getElementById("chatistyping").classList.remove("hidden");
-			if(pending_context_preinjection!=null && pending_context_preinjection!="" && pending_context_preinjection.includes(":"))
-			{
-				//use chatopponent name if we have it. do not read from settings as the inferred name may be different
-				document.getElementById("chataityping").innerText = pending_context_preinjection.split(":")[0] + " is typing...";
-			}else{
-				document.getElementById("chataityping").innerText = "The AI is typing...";
-			}
-
-		}
-
-		document.getElementById("chat_msg_send_btn").disabled = document.getElementById("btnsend").disabled;
+	function render_enhanced_chat_instruct(input)
+	{
+		var chatbody = document.getElementById("chat_msg_body");
+		if (!chatbody) { return; }
+		
+		let you = get_instruct_starttag(); let bot = get_instruct_endtag(); // Instruct tags will be used to wrap text in styled bubbles.
+		let youStyle = `</p></div></div></div><div class="incoming_msg" style='width:100%'><div class="chat_received_msg" style='width:100%'><div class="chat_received_withd_msg" style='width:100%'><p>`;
+		let botStyle = `</p></div></div></div><div class="incoming_msg" style='width:100%'><div class="chat_received_msg" style='width:100%'><div class="chat_received_withd_msg" style='width:100%;background-color:rgba(20, 20, 40, 0.4)'><p>`;
```

Comment:
![image](https://github.com/LostRuins/lite.koboldai.net/assets/39025047/8ea72935-3997-4b56-84a6-2ab482eb5e24)

I think the colors could be improved, this feels not very readable due to contrast issues.

Reply:
Text outside `seq_start` or `seq_end` (e.g.: system messages) were left faded out by design, but maybe it's not a great idea to do it like that.. hm..

There's a bug in shown in your picture where "speech" outside chat bubbles doesn't get faded out :D

I'll try something different, maybe wrapping the system message in a 'System' bubble.

Reply:
Here's how it looks with a proper background and without the fading:
![image](https://github.com/LostRuins/lite.koboldai.net/assets/32474602/d3960983-63d1-4d44-bbc8-196780e7c1b6)

Reply:
Feel free to suggest any specific colors or jump in and adjust them as needed.
I would like to add a full-on color/font customization panel eventually that opens with a click on some 🎨 button maybe, but yeah that's not for now :D